src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -26,44 +26,230 @@ export PAI_DEFAULT_FS_URI={{ hdfsUri }}
 exec 13>$LAUNCHER_LOG_DIR/YarnContainerDebug.log
 BASH_XTRACEFD=13
 
+RUNTIME_YARN_ERR_LOG=$LAUNCHER_LOG_DIR/runtime.yarn.pai.error
+RUNTIME_YARN_LOG=$LAUNCHER_LOG_DIR/runtime.yarn.pai.log
+RUNTIME_DOCKER_ERR_LOG=$LAUNCHER_LOG_DIR/runtime.docker.pai.error
+RUNTIME_DOCKER_LOG=$LAUNCHER_LOG_DIR/runtime.docker.pai.log
+USER_ERROR_FILE=$LAUNCHER_LOG_DIR/user.pai.error
+DISK_CLEANER_ERROR_FILE=$LAUNCHER_LOG_DIR/diskCleaner.pai.error
```

Comment:
Could you please first contact @wangdian to see if below path
`DISK_CLEANER_ERROR_FILE=$LAUNCHER_LOG_DIR/diskCleaner.pai.error`
can be passed to diskcleaner?

Reply:
Sure I will ask Dian to confirm

Reply:
Confirmed with Dian, this path is fine